features/having_order_partially_refunded.feature

```diff
@@ -0,0 +1,28 @@
+@refunds
+Feature: Having order partially refunded
+    In order to note that part of the order total is refunded
+    As an Administrator
+    I want to have order partially refunded
```

Comment:
It implies that the Administrator is the one that placed the order, IMO. What about `I want to be aware of the order payment state being partially refunded`?